translations/fr/messages.php

```diff
@@ -39,13 +39,18 @@
     'The alias of the plot (e.g. MAU17-PG_38_WW_1_19_5)' => 'L\'alias du plot (ex. MAU17-PG_38_WW_1_19_5)',
     'The experiment modalities of the plot (e.g. WW, WD)' => 'Les modalités expérimentales du plot (ex. WW, WD)',
+    'The object update has failed. See insertion status column for more details.' => 'La mise à jour des objets a échoué. Voir la colonne status pour plus d\'informations',
     'The replication of the plot (e.g. 2, A)' => 'La répétition du plot (ex. 2, A)',
     'The URI of the scientific object (e.g http://www.phenome-fppn.fr/phenovia/2017/o1028649)' => 'L\'URI de l\'objet scientifique (ex. http://www.phenome-fppn.fr/phenovia/2017/o1028649)',
     'The URI of the experiment (e.g. http://www.phenome-fppn.fr/pheno3c/P3C2017-6)' => 'L\'URI de l\'experimentation (ex. http://www.phenome-fppn.fr/pheno3c/P3C2017-6)',
     'The URI of the species (e.g. http://www.phenome-fppn.fr/id/species/zeamays)' => 'L\'URI de l\'espèce (ex. http://www.phenome-fppn.fr/id/species/zeamays)',
     'The error above occurred while the Web server was processing your request.' => 'L\'erreur ci-dessus est survenue lors du traitement de votre requête par le serveur web.',
     'The value' => 'La valeur',
     'The variety used in the plot (e.g. apache)' => 'La variété du plot (ex. apache)',
+    'To change the value of one property (e.g. modality), you need to provide all of its former properties that you want to keep, otherwise this information will be lost.' => 'Pour changer la valeur d\'une propriété (ex : modalité), vous devez renseigner toutes les propriété de l\'objet scientifique. Sinon, les propriétés vides seront considérées comme des valeurs à supprimer et de l\'information sera perdue.',
+    'It is not possible to change the experiment.' => 'Il n\'est pas possible de changer l\'expérimentation d\'un objet scientifique',
+    'To assign scientific objects to two experiments, you need to first create the scientific objects in one of the experiments. Then, update the scientific objects with the uri of the second experiment in the "experiment" column and their metadata in the second experiment (e.g. give the new alias and repeat the experimental modality values if they are the same).' => 'Pour assigner un objet scientifique à deux expérimentations, vous devez commencer par créer l\'objet scientifique dans une expérimentation. Mettez ensuite à jour l\'objet scientifique en renseignant l\'uri de la seconde expérimentation dans la colonne "expérimentation", en n\'oubliant pas de renseigner les métadonnées de l\'objet. Par exemple, renseignez le nouvel alias de l\'objet scientifique dans le contexte de la seconde expérimentation et répéter la valeur de la colonne modalité expérimentale qui ne change pas',
```

Comment:
'To assign scientific objects to two experiments, create the scientific objects in one of the experiments. Then, update it with the uri of the second experiment.' => 'Pour assigner un objet scientifique à deux expérimentations, créer l\'objet scientifique dans une expérimentation. Mettez ensuite à jour l\'objet scientifique en renseignant l\'uri de la seconde expérimentation.